README.md

```diff
@@ -32,36 +32,36 @@ Once you have created your Python project (e.g., `my_project`) with EspressoDB
     ## Print a formatted summary of the table entry
     print(hamiltonian)
 
-    ## Allocate an actual matrix for given entry and use it for computations
+    ## Allocate an actual matrix for the given entry and use it in computations
     eigs, vecs = np.linalg.eigh(hamiltonian.matrix)
     ```
-    Instances of a `models` class are regular classes in Python.
```

Comment:
Instances of classes are not classes, so I assume you meant either

* `models` classes are regular Python classes
* instances of a `models` class are regular Python objects

Reply:
Yes, I believe the second sentence captures the essence best.